send-email-advanced/send-email-continuation-token/send-email-continuation-token.py

```diff
@@ -0,0 +1,48 @@
+from azure.communication.email import EmailClient
+
+connection_string = "<ACS_CONNECTION_STRING>"
+sender_address = "<SENDER_EMAIL_ADDRESS>"
+recipient_address = "<RECIPIENT_EMAIL_ADDRESS>"
+
+POLLER_WAIT_TIME = 10
+MAX_POLLS = 18
+
+message = {
+    "senderAddress": sender_address,
+    "recipients":  {
+        "to": [{"address": recipient_address}],
+    },
+    "content": {
+        "subject": "Test email from Python Sample",
+        "plainText": "This is plaintext body of test email.",
+        "html": "<html><h1>This is the html body of test email.</h1></html>",
+    }
+}
+
+try:
+    client = EmailClient.from_connection_string(connection_string)
+    poller = client.begin_send(message);
+
+    # Pauses operation and saves state that can be used later to resume operation
+    token = poller.continuation_token()
+
```

Comment:
Maybe add a comment here mentioning additional processing could be done here in between after pausing and before resuming below.